src/main.py

```diff
@@ -1,7 +1,16 @@
 """The main pipeline"""
 
-from src.utils.testfunctions import add
+import toml
+from utils.testfunctions import add
 
+def toml_parser():
+    """Parse the toml config file"""
+    return toml.load("/home/cdsw/research-and-development/config/userconfig.toml")
+
+def period_select():
+    """Selects period defined by user in userconfig.toml"""
```

Comment:
Follow the [style guide](https://google.github.io/styleguide/pyguide.html) on creating docstrings. :+1: